src/docs/common-footer.adoc

```diff
@@ -17,6 +17,9 @@ if(window.location.hostname.includes("doc.activeeon.com")){
     document.querySelectorAll('.restUrl').forEach(function(element){
         element.href="https://try.activeeon.com/rest";
     });
+    document.querySelectorAll('.schedulerGraphQLUrl').forEach(function(element){
+        element.href="https://try.activeeon.com/scheduling-api/v1/graphiql";
```

Comment:
is it normal that it shows Error 500 Server Error at the right hand side?

Reply:
yes; you need to be connected first to one of the portals (studio, scheduler, etc) before being able to use it (this is explained in this documentation)